src/permissions/roles.ts

```diff
@@ -0,0 +1,71 @@
+export default {
+  GUEST: {
+    title: 'Guest',
+    permissions: [],
+  },
+  STUDENT: {
+    title: 'Student',
+    permissions: [],
+  },
+  ADMIN: {
+    title: 'Admin',
+    permissions: [
+      'dashboard.administrator.view',
+      'dashboard.profile.view',
+      'dashboard.score_report.*',
+      'dashboard.progress_report.*',
+      'dashboard.student_report.*',
+      'dashboard.admin_forms.list_orgs',
+      'dashboard.admin_forms.list_users',
```

Comment:
The current permission structure in `admin_forms` namespace seems to mix concerns. I'd recommend we follow REST-like resource naming conventions. I would recommend restructuring these permissions to be resource-centric rather than form-centric. For example: 

```suggestion
      'dashboard.organizations.list',
      'dashboard.users.list',
```